### PR TITLE
[13.0][MIG] stock_barcodes: Cherry-picked

### PR DESCRIPTION
**Pull request by olgamarcocb**
@etobella @sergio-teruel  @LoisRForgeFlow

### CODE REVIEW

**Review by etobella — APPROVED** [review]

**Review by sergio-teruel — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@olgamarcocb please rebase and check errors in Travis.

File: product_multi_barcode/models/product_product.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def _inverse_barcode(self):
         for product in self:
             if product.barcode_ids:
                 product.barcode_ids[:1].write({"name": product.barcode})
-            if not product.barcode:
+            elif not product.barcode:
```
Comment (olgamarcocb) [inline product_multi_barcode/models/product_product.py:34]: @pedrobaeza The error was created because of this change.

**Pull request merged**